### PR TITLE
Add *caw

### PR DESCRIPTION
**Pull request by Birdiest-Bird**
## About The Pull Request

This Pull Request adds the *caw emote and its sound file. This emote is crucial for birding and goes to all my fellow corvids. Caw!!!!

ATTRIBUTION:

crow.wav by vixuxx -- https://freesound.org/s/9874/ -- License: Sampling+

## Changelog

:cl:
add: *caw
sound: sound/voice/caw.ogg
/:cl:

### CODE REVIEW

File: code/modules/emotes/definitions/audible.dm [hunk 1]
```diff
@@ -270,3 +270,15 @@
 	emote_message_1p_target = "You prbt at TARGET."
 	emote_message_3p_target = "prbts at TARGET."
 	emote_sound = 'sound/voice/prbt.ogg'
+
```
Comment (Kashargul) [inline code/modules/emotes/definitions/audible.dm:273]: Needs to be moved to modular or requires //CHOMPAdd start and //CHOMPAdd end comments

File: sound/voice/bchirp.ogg
Comment (Kashargul) [inline sound/voice/bchirp.ogg:1]: Sounds should be placed in modular_chomp

File: sound/voice/caw.ogg
Comment (Kashargul) [inline sound/voice/caw.ogg:1]: Sounds should be placed in modular_chomp

**Review by Kashargul — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -186,7 +186,10 @@ var/list/_human_default_emotes = list(
 	/decl/emote/audible/wawa,
 	/decl/emote/audible/malehumanscream,
 	/decl/emote/audible/scientist/scream,
-	/decl/emote/audible/scientist/pain
+	/decl/emote/audible/scientist/pain,
+	/decl/emote/audible/caw,
+	/decl/emote/audible/birdchirp,
```
Comment (Raeschen) [inline code/modules/mob/living/carbon/human/emote.dm:191]: no comma at last entry of list

File: code/modules/mob/living/carbon/human/emote.dm [hunk 3]
```diff
@@ -335,7 +338,9 @@ var/list/_simple_mob_default_emotes = list(
 	/decl/emote/audible/mgeow,
 	/decl/emote/audible/xenogrowl,
 	/decl/emote/audible/xenohiss,
-	/decl/emote/audible/xenopurr
+	/decl/emote/audible/xenopurr,
+	/decl/emote/audible/caw,
+	/decl/emote/audible/birdchirp,
```
Comment (Raeschen) [inline code/modules/mob/living/carbon/human/emote.dm:343]: no comma at last entry of list

**Review by Raeschen — CHANGES_REQUESTED** [review]

File: modular_chomp/code/modules/emotes/definitions/audiable.dm [hunk 4]
```diff
@@ -67,6 +67,18 @@
 	emote_message_3p_target = "gwahs at TARGET."
 	emote_sound = 'modular_chomp/sound/voice/emotes/gwah.ogg'
 
+/decl/emote/audible/caw
+	key = "caw"
+	emote_message_1p = "You caw!"
+	emote_message_3p = "caws!"
+	emote_sound = 'sound/voice/caw.ogg'
```
Comment (Kashargul) [inline modular_chomp/code/modules/emotes/definitions/audiable.dm:74]: 'modular_chomp/sound/voice/caw.ogg'

File: modular_chomp/code/modules/emotes/definitions/audiable.dm [hunk 5]
```diff
@@ -67,6 +67,18 @@
...
+
+/decl/emote/audible/birdchirp
+	key = "bchirp" // Stands for Bird Chirp (because it's not a cricket chirp like *chirp, it's a bird chirp)
+	emote_message_1p = "You chirp."
+	emote_message_3p = "chirps."
+	emote_sound = 'sound/voice/bchirp.ogg'
```
Comment (Kashargul) [inline modular_chomp/code/modules/emotes/definitions/audiable.dm:80]: 'modular_chomp/sound/voice/bchirp.ogg'

File: code/modules/mob/living/carbon/human/emote.dm [hunk 6]
```diff
@@ -335,7 +337,9 @@ var/list/_simple_mob_default_emotes = list(
 	/decl/emote/audible/mgeow,
 	/decl/emote/audible/xenogrowl,
 	/decl/emote/audible/xenohiss,
-	/decl/emote/audible/xenopurr
+	/decl/emote/audible/xenopurr,
+	/decl/emote/audible/caw,
+	/decl/emote/audible/birdchirp
```
Comment (Kashargul) [inline code/modules/mob/living/carbon/human/emote.dm:342]: no longer exists

File: code/modules/mob/living/carbon/human/emote.dm [hunk 7]
```diff
@@ -186,7 +186,9 @@ var/list/_human_default_emotes = list(
 	/decl/emote/audible/wawa,
 	/decl/emote/audible/malehumanscream,
 	/decl/emote/audible/scientist/scream,
-	/decl/emote/audible/scientist/pain
+	/decl/emote/audible/scientist/pain,
+	/decl/emote/audible/caw,
+	/decl/emote/audible/birdchirp
```
Comment (Kashargul) [inline code/modules/mob/living/carbon/human/emote.dm:191]: no longer exists

**Review by Kashargul — CHANGES_REQUESTED** [review]

File: modular_chomp/code/modules/emotes/definitions/audiable.dm [hunk 8]
```diff
@@ -67,6 +67,14 @@
 	emote_message_3p_target = "gwahs at TARGET."
 	emote_sound = 'modular_chomp/sound/voice/emotes/gwah.ogg'
 
+/decl/emote/audible/caw
+	key = "caw"
+	emote_message_1p = "You caw!"
+	emote_message_3p = "caws!"
+	emote_message_1p_target = "You caw at TARGET."
+	emote_message_3p_target = "caws at TARGET."
+	emote_sound = 'modular_chomp/sound/voice/emotes/caw.ogg' // Copyright Sampling+ 1.0 Vixxus (freesound.org) for the source audio.
```
Comment (Kashargul) [inline modular_chomp/code/modules/emotes/definitions/audiable.dm:76]: are you sure that name is correct? Vixxus

Comment (Birdiest-Bird) [reply modular_chomp/code/modules/emotes/definitions/audiable.dm]: In my mind it 100% was until I read your comment, I even was about to go "Uhm, actually yeah" then double checked and went: 'v' "oh".

Sorry for making you guys check so many times, I appreciate your patience and will try to make a better work next time uvu

**Pull request merged**